Source/ZoomNet/Models/CallLog.cs

```diff
@@ -0,0 +1,280 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// A call log item.
+	/// </summary>
+	public class CallLog
```

Comment:
This should probably be renamed to `UserCallLog` to differentiate it with `AccountCallLog` otherwise it's going to be problematic when we implement the "Get account's call logs" method.

In fact, these two classes share a certain number of properties, therefore it's probably a good idea to have a `abstract CallLog` class and the UserCallLog and the AccountCallLog should derive from this base class. FYI: I did something similar with the abstract `Meeting` class and the `RecurringMeeting`, `ScheduledMeeting` and `InstantMeeting`.